sidebars.js

```diff
@@ -142,6 +142,12 @@ const sidebars = {
               id: 'versions',
               className: 'pro-feature',
             },
+            {
+              type: 'doc', 
+              label:'Live simulation',
```

Comment:
Maybe putting this link under "Design" could be confusing 🤔 Should we have a new category for "Simulate" (Mode) under "Using the Studio"? wdyt?

Reply:
> Should we have a new category for "Simulate" (Mode) under "Using the Studio"? wdyt?

100%! It’s time for a Simulate category 🥳